schema.graphql

```diff
@@ -77,4 +244,7 @@ type Settlement @entity {
   firstTradeTimestamp: BigInt!
   "Collection of trades"
   trades: [Trade!] @derivedFrom(field: "settlement")
+  ######################################################################################################################
+  # add the solver
```

Comment:
is this the sum of eachTrades?

Reply:
No, it would be the sum of the tx cost of the settlement.

One settlement is always an ethereum transaction, you will have the `gasPrice`, `gas`. Those would be nice to store too. If you multiply gasPrice and gas you get the Ether cost (`tcCostEth`). If you multiply this by the USD price, then you get the cost in USD of the transaction.